src/stages/index.js

```diff
@@ -0,0 +1,6 @@
+export * from './checkMatrixS';
+export * from './initialize';
+export * from './updateLambda';
+export * from './updateMatrixS';
+export * from './updateMatrixA';
+export * from './normalize';
```

Comment:
This file is useless. You may directly import the right util method that you need